python-package/lightgbm/engine.py

```diff
@@ -370,6 +370,8 @@ def cv(params, train_set, num_boost_round=10,
 
     if metrics:
         params.setdefault('metric', [])
+        if isinstance(params['metric'], string_type):
```

Comment:
So how can we set different loss function for a specific task ? for example binary classification.